new-site/css/style.css

```diff
@@ -245,3 +245,242 @@ textarea {
   }
 }
 
+/************************************************************************************
+MEDIA QUERIES
+*************************************************************************************/
+
+/* logo */
+#logo {
+  width: 100px;
+  height: 70px;
+  float: left;
```

Comment:
Don't use floats, this is a very old way of doing alignment. Rather use `display: flex` or `display: grid` for alignment.

Reply:
Out of curiosity, what learning resources have recommended using `float`?

Reply:
Interesting that they recommended `float`. Any chance you could share a link to the page that used `float` for alignment?